install_config/install/advanced_install.adoc

```diff
@@ -398,6 +398,69 @@ meaning that it is available for placement of new pods. See
 xref:marking-masters-as-unschedulable-nodes[Configuring Schedulability on Masters].
 |===
 
+
+
+[[configuring-cluster-preflight-checks]]
+=== Configuring Cluster Preflight Checks
+
+Preflight checks are a set of diagnostic tasks that run as part of the `OpenShift Health Checks` Ansible role.
+They run prior to an Ansible installation of OpenShift, and ensure that required inventory values are set, and identify potential issues on a host
+that can prevent or interfere with a successful OpenShift installation.
+
+An action plugin for the Health Checks role automatically discovers all checks, and executes only those selected in a play.
+Some checks have tags, which allow for groups of related checks to be run together.
+
+To disable specific preflight checks, include the variable `openshift_disable_check` with a
+comma-delimited list of check names in your inventory file. For example:
+
+----
+openshift_disable_check=memory_availability,disk_availability
+----
+
+The following table describes available preflight checks that will run before every Ansible
+installaton:
+
+
+[[preflight-checks]]
+.Preflight Checks
+[options="header"]
+|===
+
+|Check Name |Purpose
+
+|`*memory_availability*`
```

Comment:
I wish this table could be automatically generated from code... or omitted. Maintaining this up-to-date will come at a cost.